layouts/shortcodes/release.html

```diff
@@ -1,5 +1,5 @@
 <h3 style="text-align:center;">
-    <a href="#">
+    <a href="https://coherent-labs.com/get-in-touch/">

```

Comment:
Maybe it is a good idea in the future to navigate customers to developer portal or leave a link on the site telling you something like "Already registered? <a href='https://developers.coherent-labs.com'>Log in</a> to access your coherent products."